### PR TITLE
COL-1030 Looking for collaborators: front-end toggle

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/COL-1030

Display on other people's profiles comes next.

screen shot 2017-06-21 at 11 56 50 am
screen shot 2017-06-21 at 11 57 02 am

### CODE REVIEW

File: public/app/dashboard/lookingForCollaborators.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<div class="profile-looking-for-collaborators-wrapper">
+  <div data-ng-if="isMyProfile">
```
Comment (johncrossman) [inline public/app/dashboard/lookingForCollaborators.html]: @paulkerschen What if user is not a member of course?

Comment (johncrossman) [inline public/app/dashboard/lookingForCollaborators.html]: ...I suppose s/he can upload assets, etc. so...

Comment (paulkerschen) [inline public/app/dashboard/lookingForCollaborators.html]: @johncrossman It's a good question that might apply to the profile page more broadly; I think non-member admin users can also edit their personal description, etc?

**Comment by johncrossman** [comment]
Useful in looking for Mr. Goodbar. Thanks, @paulkerschen

**Pull request merged**